google/cloud/aiplatform/training_jobs.py

```diff
@@ -1653,14 +1829,35 @@ def run(
             accelerator_count (int):
                 The number of accelerators to attach to a worker replica.
             training_fraction_split (float):
-                The fraction of the input data that is to be
-                used to train the Model. This is ignored if Dataset is not provided.
+                Optional. The fraction of the input data that is to be used to train
+                the Model. This is ignored if Dataset is not provided.
             validation_fraction_split (float):
-                The fraction of the input data that is to be
-                used to validate the Model. This is ignored if Dataset is not provided.
+                Optional. The fraction of the input data that is to be used to validate
+                the Model. This is ignored if Dataset is not provided.
             test_fraction_split (float):
-                The fraction of the input data that is to be
-                used to evaluate the Model. This is ignored if Dataset is not provided.
+                Optional. The fraction of the input data that is to be used to evaluate
+                the Model. This is ignored if Dataset is not provided.
+            training_filter_split (str):
+                Optional. A filter on DataItems of the Dataset. DataItems that match
```

Comment:
It would be worth going back through the protos and ensuring any additional information that is not argument specific but split specific is surfaced like the minus sign usage here. This can only be done on public run methods.: https://github.com/googleapis/python-aiplatform/blob/master/google/cloud/aiplatform_v1/types/training_pipeline.py#L348

Reply:
Good catch. You can put it under where the `Data fraction splits` info is in the `run` method docstring.